pkg/coreos/coreos.go

```diff
@@ -29,7 +29,7 @@ const (
 
 type CoreOS interface {
 	DownloadDiskImage(releaseImage, pullSecret string) (string, error)
-	DownloadISO(releaseImage, pullSecret string) (string, error)
+	DownloadISO(releaseImage, cpuArch, pullSecret string) (string, error)
```

Comment:
We pass 3 params that are under applianceConfig. why not just pass applianceConfig and unpack it inside the function?

Reply:
Well all the signatures in this interface get only the params they need. Not sure we need the break the convention for this API. When possible, I usually prefer funcs that get only the required simple values . But we can reconsider later on of course.